src/exception.jl

```diff
@@ -28,11 +28,35 @@ struct PyError <: Exception
     # indicator.
 end
 
+function pystr_nofail(o::PyObject)
+    if ispynull(o)
+        return "NULL"
+    else
+        s = ccall((@pysym :PyObject_Str), PyPtr, (PyPtr,), o)
+        if (s == C_NULL)
+            pyerr_clear()
+            return string(PyPtr(o))
+        end
+        return convert(AbstractString, PyObject(s))
+    end
+end
+
+function Base.showerror(io::IO, e::PyError)
+    println(io, "PyError:\n", pystr_nofail(e.val))
```

Comment:
This line is crucial. `e.val` is the python exception object. And the question is how should we print it. Master does it calling `repr`. From looking at some tutorials, it seems that calling `str` is preffered. But I am by no means sure if it is the best analog of `showerror` in python. Or if it even depends on the python version, whats the preferred thing.